src/sentry/static/sentry/app/components/eventOrGroupHeader.tsx

```diff
@@ -26,7 +26,7 @@ type Props = WithRouterProps<{orgId: string}> & {
   hideLevel?: boolean;
   query?: string;
   className?: string;
-} & DefaultProps;
+} & Partial<DefaultProps>;
```

Comment:
Why this change?

Reply:
I am also having this issue in the file `components/eventOrGroupHeader.tsx`